src/sass/partials/components/_forms.scss

```diff
@@ -24,6 +24,34 @@
     }
   }
 
+  &--add {
+    @include button(setting-get('primary accent color'), setting-get('primary accent color'), setting-get('primary color'), setting-get('primary accent color'), setting-get('primary color'), setting-get('primary color'), setting-get('primary color'));
+
+    background-image: url('/images/add.svg');
```

Comment:
@Snugug So if we're going to add the `svg` to the `html` I need to do so in this file, yah?
https://github.com/punchcard-cms/content-types/blob/master/lib/form/html.js#L179

Can you do nunjucks partials in `js` files?